src/MinimalDelegation.sol

```diff
@@ -191,18 +193,41 @@ contract MinimalDelegation is
     }
 
     /// @inheritdoc ERC1271
+    /// @dev WrappedSignature is used for both NestedTypedDataSign and NestedPersonalSign signatures.
+    ///      If the caller implementing ERC-1271 is considered safe, we verify the signature over the data directly
+    /// TypedDataSign signatures are of the form: abi.encode(bytes32, bytes(`signature ‖ APP_DOMAIN_SEPARATOR ‖ contentsHash ‖ contentsDescr ‖ uint16(contentsDescr.length)`))
+    /// No extra data is needed for NestedPersonalSign signatures.
     function isValidSignature(bytes32 data, bytes calldata wrappedSignature)
```

Comment:
nit: name digest? then we can use name digest in the innermost calls to _isValidTypedDataSig / _isValidNestedPersonalSignature ?